src/app/components/turnos/gestor-agendas/pipes/botonesAgenda.pipe.ts

```diff
@@ -0,0 +1,125 @@
+import { Auth } from '@andes/auth';
+import { Pipe, PipeTransform } from '@angular/core';
+import { IAgenda } from '../../../../interfaces/turnos/IAgenda';
+
+@Pipe({
+    name: 'botonesAgenda'
+})
+export class BotonesAgendaPipe implements PipeTransform {
+    constructor(private auth: Auth) { }
+    transform(agenda: IAgenda): any {
+        const botones = {
+            editar: false,
+            suspender: false,
+            reasignar: false,
+            clonar: false,
+            sobreturno: false,
+            revision: false,
+            cargaMasiva: false
+        };
+
+        const puedeEditar = this.auth.getPermissions('turnos:agenda:puedeEditar:').length > 0;
+        const puedeClonar = this.auth.getPermissions('turnos:agenda:puedeClonar:').length > 0;
+        const puedeRevisar = this.auth.getPermissions('turnos:agenda:puedeRevision:').length > 0;
+        const puedeSuspender = this.auth.getPermissions('turnos:agenda:puedeSuspender:').length > 0;
+        const puedeDarSobreturno = this.auth.getPermissions('turnos:agenda:puedeDarSobreturno:').length > 0;
+        const puedeReasignar = this.auth.getPermissions('turnos:agenda:puedeReasignar:').length > 0;
+        const puedeCargar = this.auth.getPermissions('turnos:agenda:asignacionMasiva:').length > 0;
+
+        if (agenda) {
+            // Se puede editar sólo una agenda que esté en estado planificacion o disponible
+            if (puedeEditar && (agenda.estado === 'pendienteAsistencia' || agenda.estado === 'planificacion' || agenda.estado === 'disponible' || agenda.estado === 'publicada')) {
+                botones.editar = true;
+            }
+            // Se pueden clonar todas las agendas, ya que sólo se usa como un blueprint
+            if (puedeClonar) {
+                botones.clonar = true;
+            }
```

Comment:
O directamente hacer 
```suggestion
            botones.clonar = puedeClonar;
```
Idem para los otros casos